package.json

```diff
@@ -31,6 +31,7 @@
     "nextcloud-server": "^0.15.9",
     "nextcloud-vue": "^0.11.4",
     "nextcloud_issuetemplate_builder": "^0.1.0",
+    "printscout": "^2.0.3",
```

Comment:
new dep is fine :+1: but please use `"2.0.3"` instead so another `npm i` does not [accidentally update this package](https://stackoverflow.com/questions/45022048/why-does-npm-install-rewrite-package-lock-json)

Reply:
fixed, thanks